NativeUI/UIMenu.cs

```diff
@@ -1447,6 +1477,15 @@ protected virtual void ItemSelect(UIMenuItem selecteditem, int index)
             OnItemSelect?.Invoke(this, selecteditem, index);
         }
 
+        protected virtual void SliderChange(UIMenuSliderItem sender, int newindex)
+        {
+            OnSliderChange?.Invoke(this, sender, newindex);
+        }
+
+        protected virtual void SliderSelect(UIMenuSliderItem sender, int newindex)
```

Comment:
This is never called

Reply:
There is UIMenuItem.Activated for that. It has been removed on 9f30dbbd272bce37699873d5d4019b99176603d6.